pkg/istio-agent/sds-agent.go

```diff
@@ -128,7 +131,7 @@ const (
 
 var (
 	// LocalSDS is the location of the in-process SDS server - must be in a writeable dir.
-	LocalSDS = "/etc/istio/proxy/SDS"
+	LocalSDS = "./etc/istio/proxy/SDS"
```

Comment:
Why change this in this pr?

Reply:
Same as above - running istiod and agent in intellij as regular user, no access to /etc.